logger/src/args.rs

```diff
@@ -9,9 +9,9 @@ pub struct Args {
     #[arg(long, default_value = "127.0.0.1:8000")]
     pub address: SocketAddr,
 
-    /// Where to store resources state
+    /// Remote database connection url
     #[arg(long, default_value = "./")]
-    pub state: PathBuf,
+    pub db_connection_uri: Uri,
```

Comment:
Maybe we should remove the default value?

Reply:
Agreed.